pilot/pkg/kube/inject/webhook.go

```diff
@@ -139,7 +139,10 @@ func NewWebhook(configFile, meshFile, certFile, keyFile string, port int) (*Webh
 		meshFile:               meshFile,
 		watcher:                watcher,
 	}
-	http.HandleFunc("/inject", wh.serveInject)
+	h := http.NewServeMux()
```

Comment:
Do we do this to capture more metrics? I'm not sure what the desired change is here.

Reply:
The original code panic'd when I added benchmarks because the default ServeMux is shared across tests.